data/json/terrain.json

```diff
@@ -192,6 +201,7 @@
         "type" : "terrain",
         "id" : "t_thconc_floor",
         "name": "concrete floor",
+        "description": "A bare and cold concrete floor with matching roof, could still insulate from the outdoors but roof collapse is possible if supporting walls are broken down.",
```

Comment:
Not sure if the mention of the roof is needed here

Reply:
Since roofs are represented with/as floors, it's as I said in the beginning:

>I did my best to make it both non-specific but specific enough to convey appearance.